src/components/Search/SearchPageHeader.tsx

```diff
@@ -43,35 +60,64 @@ function SearchPageHeader({query, selectedItems = {}, hash, clearSelectedItems,
         finished: {icon: Illustrations.CheckmarkCircle, title: translate('common.finished')},
     };
 
-    const selectedItemsKeys = Object.keys(selectedItems ?? []);
+    const selectedTransactionsKeys = Object.keys(selectedTransactions ?? []);
 
     const headerButtonsOptions = useMemo(() => {
+        if (selectedTransactionsKeys.length === 0) {
+            return [];
+        }
+
         const options: Array<DropdownOption<SearchHeaderOptionValue>> = [];
 
-        if (selectedItemsKeys.length === 0) {
-            return options;
+        // Because of some problems with the lib we use for download on native we are only enabling download for web, we should remove the SearchActionOptionsUtils files when https://github.com/Expensify/App/issues/45511 is done
```

Comment:
Let's create an issue for this if we don't already have one

Reply:
We do, it is linked in the comment https://github.com/Expensify/App/issues/45511